packages/xstate-graph/src/graph.ts

```diff
@@ -58,7 +58,7 @@ export function deserializeEventString<TEvent extends EventObject>(
 }
 
 export interface ValueAdjMapOptions<TContext, TEvent extends EventObject> {
-  events: { [K in TEvent['type']]: Array<TEvent & { type: K }> };
+  events: { [K in TEvent['type']]?: Array<TEvent & { type: K }> };
```

Comment:
im not 100% sure about this change, please check if its correct

Reply:
Not every event needs to be represented in `events`. Previously, compilation would fail if you were missing events I believe.